helm/datalab/templates/configmaps/oidc-configmap.template.yml

```diff
@@ -6,24 +6,32 @@ metadata:
 data:
   config: |
     {
-      "client_id": "{{ .Values.oidcProviderClientId }}",
-      "redirect_uri": "https://{{ .Values.datalabName }}.{{ .Values.domain }}/callback",
-      "response_type": "code",
-      "scope": "openid profile",
-      "authority": "https://mjbr.eu.auth0.com",
-      "automaticSilentRenew": true,
-      "accessTokenExpiringNotificationTime": "600",
-      "filterProtocolClaims": true,
-      "loadUserInfo": true,
-      "extraQueryParams": {
-          "audience": "https://datalab.datalabs.nerc.ac.uk/api"
+      "oidc": {
+        "userManager": {
+          "client_id": "{{ .Values.oidcProviderClientId }}",
+          "redirect_uri": "https://{{ .Values.datalabName }}.{{ .Values.domain }}/callback",
+          "response_type": "code",
+          "scope": "openid profile",
+          "authority": "https://mjbr.eu.auth0.com",
+          "automaticSilentRenew": true,
+          "accessTokenExpiringNotificationTime": "600",
+          "filterProtocolClaims": true,
+          "loadUserInfo": true,
+          "extraQueryParams": {
+            "audience": "{{ .Values.oidcProviderAudience }}"
```

Comment:
I think a value needs to be added for this variable to `helm/datalab/values.yaml` such that there is a default value for when we are deploying to test. This is how the other values have been configured. I then foresee us having different value files for production etc. like we did when using `bara`.

Reply:
It already exists there, see https://github.com/NERC-CEH/datalab-k8s-manifests/blob/master/helm/datalab/values.yaml#L20.  This was a quick fix because I'd spotted it wasn't being used in the template.

Reply:
Ahhh I see; sorry I should have had a better look in the values file! Well spotted!